Log Viewing/ARKLogTableViewController.m

```diff
@@ -503,28 +499,22 @@ + (NSDateFormatter *)sharedDateFormatter;
 
 #pragma mark - Initialization
 
-- (instancetype)initWithDate:(NSDate *)date;
+- (instancetype)init;
```

Comment:
will mark as NS_UNAVAIL, but still need this in case of performSelector: shenanigans